### ISSUE TITLE
Hash speed drops

### ISSUE THREAD

**Issue by Valtsuh** [xmrig/xmrig#2480]
Hello, I'm not really sure to as to where to post, but my hash speed seems to just drop overtime, eventually going all the way down to under 1h/s (from 400 ish / thread (laptop), max. roughly 1200 with 3-4 threads). 

Any help?

**Comment by SChernykh** [comment]
https://xmrig.com/docs/miner/randomx-stability-troubleshooting-guide

**Comment by Valtsuh** [comment]
> https://xmrig.com/docs/miner/randomx-stability-troubleshooting-guide

> The only known workaround for this is to unpin threads from cores

After just opening the config file for the miners, 

>was changed, reloading configuration

Although the config file was only opened (maybe a quick tweak?), opening the file isn't exactly modifying the file, anyway, the speed does seem to bump up, maybe the miner isn't refreshing itself as it should, certainly would seem like it?
![detox xmrig speed](https://user-images.githubusercontent.com/77432892/125094267-ca371800-e0db-11eb-8f96-2c1e08cd3503.jpg)

I tried quick unpinning, pinning, but just refreshing the config seemed to work the most. I'll try again once I notice the speed dropping (if drops) down to near 0.

**Comment by SChernykh** [comment]
You didn't read the full guide, better workaround is to turn off memory compression and RunFullMemoryDiagnostic tasks.

**Comment by Valtsuh** [comment]
Yeah I don't know if doing that made any difference, the speed just seems wavy, doesn't seem to stabilize at all.

**Comment by Valtsuh** [comment]
Been running a few hours now, hashrate very low, with the above mentioned advice.

**Comment by Shai0Hulud** [comment]
It's a laptop? I would guess that Overheat protection kicks in. 

And depending on electricity costs, not sure if it's really worth it with a hashrate like this

**Comment by Valtsuh** [comment]
> It's a laptop? I would guess that Overheat protection kicks in. 
> 
> And depending on electricity costs, not sure if it's really worth it with a hashrate like this

I'm not too worried 'bout electricity costs, thought about getting an ant. If I can get to withdrawing the coins i'm fine with it. 400(BTT) limit at unmineable. GAS running aswell, seema a lot for just one coin. Considering the site bases on rewarding as they mention "mining". Not sure if the reward rate increases or decreases with hashrate, seems constant.
